source/_components/device_tracker.ubee.markdown

```diff
@@ -40,5 +45,13 @@ password:
   type: string
 {% endconfiguration %}
 
+Supported models:
+- EVW32C-0N
+- EVW3200-Wifi
+
+<p class='note info'>
+This component uses [pyUbee library](https://github.com/mzdrale/pyubee), which was tested with models listed above. If you have different model of [Ubee Router](http://www.ubeeinteractive.com/products) and want to use this component, try specifying one of models listed above, it might work. If that happens, please [create pyUbee issue](https://github.com/mzdrale/pyubee/issues/new) to report. Also if none of these models work for you, [create pyUbee issue](https://github.com/mzdrale/pyubee/issues/new) to request for support for your model.
```

Comment:
You must put the links in html style 😉 https://www.w3schools.com/html/html_links.asp

Reply:
My bad, I was in hurry and didn't see I mixed html and markdown. :)